defender/refill-tokens/autotasks/refill-tokens/index.js

```diff
@@ -1,58 +1,63 @@
 const { DefenderRelaySigner, DefenderRelayProvider } = require('defender-relay-client/lib/ethers');
 const ethers = require('ethers');
-const TokenTypeEnum = { ERC20: 0, ERC721: 1, ERC1155: 2 };
-const TOKEN_TYPE = TokenTypeEnum.ERC20;
-const ERC1155ID = '1';
-const TOKEN_ADDRESS = ethers.constants.AddressZero;
-const REFILL_VALUE = '1';
-const REFILL_ADDRESS = ethers.constants.AddressZero;
-const FILLED_VALUE = new ethers.BigNumber.from('3');
-const ERC1155Abi = require('../../../../abi/contracts/ERC1155Mock.sol/ERC1155Mock.json');
-const ERC721Abi = require('../../../../abi/contracts/ERC721Mock.sol/ERC721Mock.json');
-const ERC20Abi = require('../../../../abi/contracts/ERC20Mock.sol/ERC20Mock.json');
+// const tokenAddress = ethers.constants.AddressZero;
+const recipientTopUpBalance = '1';
+// const recipientAddress = ethers.constants.AddressZero;
+const recipientTopUpAmount = new ethers.BigNumber.from('3');
+
+import {
+  tokenType, nftId,
+  'token-address' as tokenAddress,
+  'recipient-address' as recipientAddress
+} from '../../config.dev.yml';
```

Comment:
If we want to import directly from the config file, I think it needs to be in camelcase, or it'll yell at us. My suggestion would be to just change to camelcase and then change the names in the `config.dev.yml` file

```suggestion
import {
  tokenType, 
  nftId,
  tokenAddress,
  recipientAddress,
} from '../../config.dev.yml';
```

Reply:
it goes against our conventions with the other templates, but it seems like a clean way to do it to avoid uploading them as a secret to defender

Reply:
Switched it to Autotask secrets